recipes/enable-postfix-smarthost.rb

```diff
@@ -30,13 +34,6 @@
   })
 end
 
-file '/etc/mailname' do
-  mode '600'
-  owner 'root'
-  group 'root'
-  content "#{hostname}.localdomain\n"
-end
-
```

Comment:
`/etc/mailname` is a debian/ubuntu thing. We'll configure the postfix mail_name directly in the postfix config.